### DISCUSSION TITLE
Tikmanager for Unreal

### DISCUSSION THREAD

**Discussion by bodibod** [masqu3rad3/tik_manager4#141] [Ideas]
Hey Arda, are you planning to integrate Unreal in the near future ?

**Comment by masqu3rad3** [comment]
Hey Tanguy, that is among my plans but not the top priority at this point. One reason for that is that Unreal works is totally different and unsuitable for a similar version control mechanism we apply for other DCCs. (I am having the same issue with Davinci Resolve)
Therefore I need to come up with a different strategy for UE and it won't be inline with the rest of DCCs.

I would appreciate any thought on this matter such as what you would expect Tik Manager to do and how to interact with Unreal.

My current prio is integrating project management softwares such as Shotgrid and Kitsu by the way.
